pkg/cmd/cli/cmd/set/env.go

```diff
@@ -48,6 +48,12 @@ syntax.`
   # Update all containers in all replication controllers in the project to have ENV=prod
   %[1]s env rc --all ENV=prod
 
+  # Import environment from a secret
+  %[1]s env --from-secret=mysecret dc/myapp
```

Comment:
What if they need to be?

On Jun 7, 2016, at 7:54 PM, Jordan Liggitt <notifications@github.com> wrote:

In pkg/cmd/cli/cmd/set/env.go
<https://github.com/openshift/origin/pull/9210#discussion_r66173026>:

> @@ -48,6 +48,12 @@ syntax.`
>    # Update all containers in all replication controllers in the project to have ENV=prod
>    %[1]s env rc --all ENV=prod
>
> +  # Import environment from a secret
> +  %[1]s env --from-secret=mysecret dc/myapp

Secrets aren't injected as base64 are they?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/9210/files/2dca495d9d658651af8fa61da45494ba31c65f6f#r66173026>,
or mute the thread
<https://github.com/notifications/unsubscribe/ABG_p-oAqTxOUeTldM7Xie2jTogymy0sks5qJgSagaJpZM4IwT6g>
.